### PR TITLE
implement authentication

### PR DESCRIPTION
**Pull request by anmolsingh0001**
implement authentication 
Thanks @2005balaji for making this PR possible.

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+    if(!check){
+        return  {status:"Please check email and password",user:null}
+    }
+
+    if(ispasswordvalid){
+        const token=jwd.sign({
+            email:check.email,
+            password:check.password
+        },'n8tv3222hk')
```
Comment (aditya-190) [inline backend/server.js]: Please add this private key inside your .env file and use from there.

File: frontend/src/Components/Navbar.js [hunk 2]
```diff
@@ -61,62 +54,178 @@ function Navbar() {
...
-              </a>
+              
+              </Link>
             </div>
           </nav>
         </div>
+
+        {
+
+        (inputvalu)?
```
Comment (aditya-190) [inline frontend/src/Components/Navbar.js]: rename this variable to `inputValue`

**Review by aditya-190 — CHANGES_REQUESTED** [review]
Pls make the required changes before merging.

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge pull request.

**Pull request merged**